zkvm-prover/release-verifier-stuff.sh

```
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# release version
+SCROLL_ZKVM_STUFFDIR ?= `realpath .work`
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)